src/locale/nn.yml

```diff
@@ -152,7 +152,7 @@ feat:
         remove: Fjern fra oppdrag
       add:
         alreadyAdded: Lagt til tidligere
-        placeholder: Legg til ringer
+        placeholder: Begynn å skrive for å søke eller legge til en ny oppringer
```

Comment:
someone should verify, weather this translation is accurate (this is norsk bokmal)